Code/GraphMol/ChemReactions/testReaction.cpp

```diff
@@ -6878,14 +6925,258 @@ void testGithub1868() {
   }
 }
 
+bool check_bond_stereo(const ROMOL_SPTR &mol, unsigned bond_idx,
+                       int start_anchor_idx, int end_anchor_idx,
+                       Bond::BondStereo stereo) {
+  auto *bond = mol->getBondWithIdx(bond_idx);
+  std::vector<int> stereo_atoms_ref{{start_anchor_idx, end_anchor_idx}};
+  return Bond::BondType::DOUBLE == bond->getBondType() &&
+         stereo == bond->getStereo() &&
+         stereo_atoms_ref == bond->getStereoAtoms();
+}
+
+ROMOL_SPTR run_simple_reaction(const std::string &reaction,
+                               const ROMOL_SPTR &reactant) {
+  const auto rxn = RxnSmartsToChemicalReaction(reaction);
+  TEST_ASSERT(rxn);
+  TEST_ASSERT(rxn->getNumReactantTemplates() == 1);
+  TEST_ASSERT(rxn->getNumProductTemplates() == 1);
+  rxn->initReactantMatchers();
+
+  MOL_SPTR_VECT reacts{{reactant}};
+  const auto prods = rxn->runReactants(reacts);
```

Comment:
I think there's also a `rxn->runReactant(reactant)`

Reply:
It is possible, I added some tests before looking to much into this, and then never checked back.